### PR TITLE
tooltip docs

### PR DESCRIPTION
**Pull request by danethurber**
toolips ya'll

### CODE REVIEW

**Review by jaketrent — APPROVED** [review]
Interested in your take on the organizational issues I brought up here.

File: docs/src/components/component.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-export { Guideline } from './guideline'
```
[the remark this answers is not in the document]
Comment (danethurber) [reply docs/src/components/component.ts:1]: i can get behind that. do you think the `Usage` and `Intro` components should be part of that single import?

File: docs/src/components/guidelines/guidelines.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ interface GuidelineProps {
   dont: React.ReactNode
   dontStyle: Record<string, unknown>
 }
-export const Guideline: React.FC<GuidelineProps> = props => {
+export const Guidelines: React.FC<GuidelineProps> = props => {
```
[the remark this answers is not in the document]
Comment (danethurber) [reply docs/src/components/guidelines/guidelines.tsx:16]: i can update

File: docs/src/pages/components/tooltip.mdx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+            appearance={Button.appearances.secondary}
+            onClick={() => setClicked(!isClicked)}
+          >
+            Click me
+          </Button>
+        </Below>
+      </div>
+
+      <style jsx>{`
+        .examples { display: flex; padding: ${layout.spacingLarge}; }
```
[the remark this answers is not in the document]
Comment (danethurber) [reply docs/src/pages/components/tooltip.mdx:70]: i prefer that is was copy/pasta-ed from the old docs 😄 - i'll see about using one of the other example classes

[the remark this answers is not in the document]
Comment (jaketrent) [reply docs/src/pages/components/tooltip.mdx:70]: I was just curious.

I think I may actually prefer this.  It's less opaque, thought a little bit more busy. styled-jsx puts it in its own style block. Also nice, it'll work on codesandbox.

File: docs/src/components/component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jaketrent) [reply docs/src/components/component.ts:1]: Yes, I think so. 

They already are, btw:
https://github.com/pluralsight/design-system/pull/1199/files/d66df978566e9bf593216e68b9efee406fff5ce0#diff-77e92f23d9b8fe75c29b7a5447121591

**Pull request merged**